api/src/backend/api/filters.py

```diff
@@ -704,3 +705,16 @@ class Meta:
         fields = {
             "inserted_at": ["date", "gte", "lte"],
         }
+
+
+class LighthouseConfigFilter(FilterSet):
+    inserted_at = DateFilter(field_name="inserted_at", lookup_expr="date")
+
+    class Meta:
+        model = LighthouseConfiguration
+        fields = {
+            "name": ["exact", "icontains"],
+            "model": ["exact", "icontains"],
+            "is_active": ["exact"],
+            "inserted_at": ["gte", "lte"],
+        }
```

Comment:
Does this make sense if we can only save one LighthouseConfiguration per tenant? I don't feel like this endpoint should have any filter.